src/Hazelcast.Net/CP/AtomicReference.cs

```diff
@@ -35,7 +35,7 @@ internal class AtomicReference<T>: CPDistributedObjectBase, IAtomicReference<T>
         /// <param name="cluster">The cluster.</param>
         /// <param name="serializationService">The serialization service.</param>
         public AtomicReference(string name, CPGroupId groupId, Cluster cluster, SerializationService serializationService)
-            : base(ServiceNames.AtomicRef, name, groupId, cluster)
+            : base(ServiceNames.AtomicRef, name, groupId, cluster, serializationService)
         {
             SerializationService = serializationService ?? throw new ArgumentNullException(nameof(serializationService));
```

Comment:
At first look this is confusing because we assume that `SerializationService = serializationService` is performed already by the base class... Maybe I'd change it with `if (serializationService is null) throw ...` to avoid this confusion.

Reply:
This is a leftover which is fixed at 1e035072a014026fe3dd6f8b5a3f8970f4faf5a2. Base class has serialization service.